tests/console/snapper_jeos_cli.pm

```diff
@@ -28,7 +28,7 @@ sub run {
     my $openqalatest = 'openqalatest';
     assert_script_run("snapper create -d $openqalatest");
     assert_script_run("snapper list");
-    my $snap_id             = sle_version_at_least('15') ? "'{ print \$1 }'" : "'{ print \$3 }'";
+    my $snap_id             = is_sle('15+') ? "'{ print \$1 }'" : "'{ print \$3 }'";
```

Comment:
@mnowaksuse @ccret this might be a candidate for failure as I assume there is a openSUSE >= 15 jeos version where previously "sle_version_at_least('15')" might have returned true and now `is_sle` would not anymore